api/controllers/case.js

```diff
@@ -68,7 +68,7 @@ async function postCaseNewHttp(req, res) {
     let title = req.body.title;
     let body = req.body.body || req.body.summary || "";
     let description = req.body.description;
-    let language = req.body.original_language || "en";
```

Comment:
the `original_language` field  is sent up from the client as part of the body, and not as a param.

Reply:
Good catch. I make sure to get original_language from the body now and also clarified places where we use original_language vs. language.